dts/bindings/input/espressif,touch-sensor.yaml

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2023 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Zephyr input touch sensor parent node
+
+  This defines a group of touch sensors that can generate input events. Each touch
+  sensor is defined in a child node of the touch-sensor node and defines a specific key
+  code.
+
+  For example:
+
+  #include "../modules/hal/espressif/components/hal/include/hal/touch_sensor_types.h"
```

Comment:
does this work? pretty sure you can't use enums in DT

make a file in `include/zephyr/dt-bindings/input` instead and add plain defines there

Reply:
It's really a better way.